contrib/!lang/c-c++/packages.el

```diff
@@ -43,11 +44,18 @@
       (c-toggle-auto-newline 1)
       (evil-leader/set-key-for-mode 'c-mode
         "mga" 'projectile-find-other-file
+        "md" 'disaster
         "mgA" 'projectile-find-other-file-other-window)
       (evil-leader/set-key-for-mode 'c++-mode
         "mga" 'projectile-find-other-file
+        "md" 'disaster
         "mgA" 'projectile-find-other-file-other-window))))
 
+(defun c-c++/init-disaster ()
+  (use-package disaster
+    :defer t
+    :commands (disaster)))
```

Comment:
It would be better for the keybindings to be set here to keep the package config seperated properly